ci/daint-mc_test.sbatch

```diff
@@ -7,24 +7,59 @@
 
 set -e
 
+# Utilities for timers
+## current time in seconds
+function ct { date +"%s"; }
+## et(t) elapsed time since instant t (got with ct command)
+function et { date +%T -d "1/1 + $(( `ct` - $1 )) sec"; }
+
 source ${ENV}
 module list
 
-set -x
 DIR=build_${BUILD_TYPE}
 
 cd $DIR
 
 # make sure no test log exists
+rm -f Testing/Temporary/Test.log
 rm -f Testing/Temporary/LastTest.log*
 
 set +e
-make test
-RETCODE=$?
+
+RETCODE=0
+FULL_START=$(ct)
+# Allow 1-99 ranks
+for label in `ctest --print-labels | egrep -o "RANK_[1-9][0-9]?"`
+do
+  N=`echo $label | sed "s/RANK_//"`
+  C=$(( CORES_PER_NODE / N ))
```

Comment:
in which sense?
to have at least 1 or to prevent execution?

Reply:
Because few lines before you are saying that it is open for ranks from 1 to 99, but if you pass a number greater than CORES_PER_NODE you will get a 0.

I would add a check to not allow the run and signal it in some way.

Reply:
if it is 0 `srun` stop the execution already.